pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -945,8 +949,6 @@ func (m *ManagerImpl) GetDeviceRunContainerOptions(pod *v1.Pod, container *v1.Co
 			return nil, err
 		}
 	}
-	m.mutex.Lock()
```

Comment:
just recording we appear to move this to an RLock inside poddevices.

Reply:
ack, missed in my rebase. Needs to be checked.